From: Dylan Maxwell <maxwelld@frib.msu.edu>
Date: Fri, 30 Nov 2018 11:40:00 -0500
Subject: adjust setup.py

---
 setup.py | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/setup.py b/setup.py
index 929f128..f147918 100755
--- a/setup.py
+++ b/setup.py
@@ -10,9 +10,6 @@ try:
     from setuptools import setup, Extension
 
     setup_args = dict(
-        entry_points = {
-            'console_scripts': [
-                'pvtree.py = cothread.tools.pvtree:main' ] },
         install_requires = ['numpy'],
         zip_safe = False)
 
@@ -47,8 +44,10 @@ extra_compile_args = [
     '-Wno-unused-parameter',
     '-Wno-missing-field-initializers',
     '-Wundef',
+    '-Wshadow',
     '-Wcast-align',
     '-Wwrite-strings',
+    '-Wredundant-decls',
     '-Wmissing-prototypes',
     '-Wmissing-declarations',
     '-Wstrict-prototypes']
@@ -66,7 +65,7 @@ if platform.system() == 'Windows':
 
 setup(
     name = 'cothread',
-    version = get_version(),
+    version = '2.15',
     description = 'Cooperative threading based utilities',
     long_description = open('README.rst').read(),
     author = 'Michael Abbott',
@@ -75,7 +74,6 @@ setup(
     license = 'GPL2',
     packages = ['cothread', 'cothread.tools'],
     ext_modules = ext_modules,
-    test_suite="tests",
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
